test/test_lazy.py

```diff
@@ -0,0 +1,95 @@
+import torch
+from common_utils import (
+    TestCase,
+    run_tests,
+)
+import time
+
+
+class TestLazy(TestCase):
+    def test_bench(self):
+        a = torch.rand(1, 1)
+        b = torch.rand(1, 1)
+        a_ = a.to_lazy()
+        b_ = b.to_lazy()
+        iters = int(1e2)
+
+        t = time.time()
+        for _ in range(iters):
+            c = a + b
+            a = c
+        print(time.time() - t)
+
+        t = time.time()
+        for _ in range(iters):
+            c_ = a_ + b_
+            a_ = c_
+        print(time.time() - t)
+        c_ = c_.to_eager()
+        torch.testing.assert_allclose(c_, c)
+
+    def test_basic(self):
+        a = torch.rand(10, 10)
+        b = torch.rand(10, 10)
+        c = a
+        d = b
+
+        a = a.to_lazy()
```

Comment:
Hi, I have been trying to do something pretty similar to what you are doing, however I cant get "is_lazy" to work in python side... Did you guys have any problem with that?

Reply:
Hmm, that's not enough detail to really say what the problem might be.

Reply:
I just assume there is some problem dealing with a function returning a single boolean... Maybe a better question is, where is is_sparse in python registered? Maybe I should register it the same way as is_sparse.

Reply:
IIRC `is_sparse` is handled specially. See `torch/csrc/autograd/python_variable.cpp`